app/question.js

```diff
@@ -1,8 +1,13 @@
 define(['jquery', 'app/messages', 'app/config', 'app/string'], function ($, messages, config, string) {
-  var currentItemIndex = 0;
-  var inTutorial = config.constant("TUTORIAL_MODE");
   var items;
+  var currentItemIndex = 0;
   var totalLength;
+  
+  var itemsAnsweredCorrectly = 0;
+  var answerWasCorrect;
+  
+  var tutorialLength;
+  var inTutorial = config.constant("TUTORIAL_MODE");
 
   // Calculate the percentage of 'part out of total'
   function percentage(part, total) {
```

Comment:
Shouldn't this function be in some kind of  math module? Or do you really only need it here?